### PR TITLE
site: simplify instructions to find service external IP

### PR DESCRIPTION
**Pull request by medyagh**
instead of telling the user to examine output of a command for external IP, use a better command:
tested locally
```
$ curl $(kubectl get services discover2 --output=jsonpath='{.status.loadBalancer.ingress[0].ip}'):8080/version
version: 0.0.1

```

### CODE REVIEW

**Comment by medyagh** [comment]
doing /ok-to-test to observe the current test failures on the master

File: site/content/en/docs/start/_index.md [hunk 1]
```diff
@@ -150,9 +150,11 @@ In another window, start the tunnel to create a routable IP for the 'balanced' d
 minikube tunnel
 ```
 
-To find the routable IP, run this command and examine the `EXTERNAL-IP` column:
+To find the routable external IP, run this command:
 
-`kubectl get services balanced`
+```shell
+kubectl get services balanced --output=jsonpath='{.status.loadBalancer.ingress[0].ip}''
```
Comment (tstromberg) [inline site/content/en/docs/start/_index.md:156]: I would rather not: I really do not expect folks to be able to memorize this command.

Comment (medyagh) [reply site/content/en/docs/start/_index.md:156]: sounds reasonable. will close

**Pull request closed without merging**